guibot/finder.py

```diff
@@ -138,7 +138,7 @@ def from_string(raw):
         :raises: :py:class:`ValueError` if unsupported type is encountered
         """
         args = []
-        string_args = re.match(r"<value='(.+)' min='(-?[\d.None]+)' max='([\d.None]+)'"
+        string_args = re.match(r"<value='(.*)' min='(-?[\d.None]+)' max='([\d.None]+)'"
```

Comment:
Also a really well focused fix, would you mind adding a unit test like
```diff
----------------------------- tests/test_finder.py -----------------------------
index fa443c7..7fc8475 100644
@@ -1052,7 +1052,17 @@ class CVParameterTest(unittest.TestCase):
             tolerance=10.2, fixed=False, enumerated=False
         )
         parsed = CVParameter.from_string("<value='123456789.' min='None' max='None' delta='1030.25' tolerance='10.2' fixed='False' enumerated='False'>")
         self.assertEqual(parsed, expected)
 
+    def test_empty_value(self):
+        """Check that the parser handles empty CVParameter value gracefully."""
+        expected = CVParameter(
+            "", min_val=None, max_val=None, delta=10.0,
+            tolerance=1.0, fixed=True, enumerated=False
+        )
+        parsed = CVParameter.from_string("<value='' min='None' max='None' delta='10.0' tolerance='1.0' fixed='True' enumerated='False'>")
+        self.assertEqual(parsed, expected)
+
+
 if __name__ == '__main__':
     unittest.main()
```
From what I can see it also detects the problem as
```
======================================================================
ERROR: test_empty_value (__main__.CVParameterTest)
Check that the parser handles empty CVParameter value gracefully.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/guibot/tests/test_finder.py", line 1063, in test_empty_value
    parsed = CVParameter.from_string("<value='' min='None' max='None' delta='10.0' tolerance='1.0' fixed='True' enumerated='False'>")
  File "/usr/lib/python3.10/site-packages/guibot/finder.py", line 143, in from_string
    raw).group(1, 2, 3, 4, 5, 6)
AttributeError: 'NoneType' object has no attribute 'group'
```
and would be beneficial for future coverage.